rust/headless-client/src/lib.rs

```diff
@@ -562,12 +564,9 @@ mod tests {
     fn cli() -> anyhow::Result<()> {
         let exe_name = "firezone-headless-client";
 
-        let actual = Cli::parse_from([exe_name]);
-        assert_eq!(actual.api_url, Url::parse("wss://api.firezone.dev")?);
-        assert!(!actual.check);
```

Comment:
Not related, I just happened to hit this. Running `cargo test` in dev mode uses the staging API by default. I don't remember if that was behavior from the original `linux-client` or if I added it, but it made this test complicated, so I just got rid of it. All it does is double-check that `clap` does what I want anyway.